var/spack/repos/builtin/packages/trinotate/package.py

```diff
@@ -35,6 +34,14 @@ def patch(self):
             perlscripts = glob.glob('*.pl')
             filter_file('#!/usr/bin/perl', '#!/usr/bin/env perl', *perlscripts)
 
+        # trinotate web generates a config on run but puts it in a bad place
+        # this causes issues with permissions; we hack the source to keep it
+        # in the calling user's homedir
```

Comment:
why do you put it in HOME? Shouldn't it got in PREFIX

Reply:
on a shared spack installation users normally won't have write access to the spack prefix